### ISSUE TITLE
Slash Commands

### ISSUE THREAD

**Issue by bfzdk** [iBlitzkriegi/Vixio#341]
Please add support for Slash Commands!

**Comment by NoboKik** [comment]
Use DiSKy - https://disky.itsthesky.info/

**Comment by Olyno** [comment]
@NoboKik This is not the point, and not a good answer. The suggestion is to add slash command to vixio, not switching of addon.

**Comment by CerialPvP** [comment]
> @NoboKik This is not the point, and not a good answer. The suggestion is to add slash command to vixio, not switching of addon.

you're right but the addon is not gonna get updated for another 1 trillion years so you will be better off using a different addon.

**Comment by Olyno** [comment]
Again, this is not the point here. If @Fr3ckz wants to change his addon, so he gonna ask for it. This is not what he asked for.

**Comment by goldfinder** [comment]
> @NoboKik This is not the point, and not a good answer. The suggestion is to add slash command to vixio, not switching of addon.

Unfortunately this plugin is no longer in development.  The last published version was 2 YEARS AGO.
